definitions/divorce/json/CaseEventToFields.json

```diff
@@ -11328,5 +11328,389 @@
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-AdulteryCoResp-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolSOCCoRespondent",
+    "PageLabel": "Statement of case - adultery",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-Behaviour-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolSOCBehaviour1",
+    "PageLabel": "Statement of case - behaviour",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-LivedApart-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolSOCLivedApart",
+    "PageLabel": "Lived Apart for the entire time",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-LegalProceedings-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolExistingCourtCases",
+    "PageLabel": "Other legal proceedings",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-FinancialOrders-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolDividingMoneyAndProperty",
+    "PageLabel": "Financial orders",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-ClaimCosts-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolApplyToClaimCosts",
+    "PageLabel": "Claim for costs",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-UploadMarriageCert-Create",
+    "PageFieldDisplayOrder": 0,
+    "DisplayContext": "READONLY",
+    "PageID": "SolUploadDocs",
+    "PageLabel": "Upload the marriage certificate",
+    "PageDisplayOrder": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "07/04/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorUpdate",
+    "CaseFieldID": "LabelSolAboutEditingApplication-AboutSolicitor-Update",
```

Comment:
I think you could have re-used the same fields for the 'solicitorCreate' and 'solicitorUpdate' events right? as they are different events, you wouldn't have the error of 'same eventId + caseFieldId'
Have you tried that?
so for instance, you could re-use: LabelSolAboutEditingApplication-AboutSolicitor-Create on 'solicitorCreate' event and on the 'solicitorUpdate' event. So you would have 16 fields rather than 32 I think

Reply:
Good spot, i've tested and applied those changes 👍